tests/testthat/test-prepare_matches_mapper.R

```diff
@@ -7,3 +7,31 @@ test_that("`completion` column has either `low`, `medium`, or `high` values", {
   out <- prepare_matches_mapper(small_matches_mapper, ecoinvent_activities)
   expect_equal(unique(out$completion), c("low", "high", "medium"))
 })
+
+test_that("`prepare_matches_mapper` doesn't have any empty string", {
+  tibble_names <- function(x, nms) {
+    m <- matrix(x, ncol = length(nms), dimnames = list(NULL, nms))
+    tibble::as_tibble(m)
+  }
```

Comment:
This helper is already useful in 2 tests, and maybe more. So I think it's best to move it to R/utils-tests.R (or tests/testthat/helper.R but R/utils-tests.R is more discoverable).

Reply:
Added to R/utils-tests.R